src/Internal/QueryM/Kupo.purs

```diff
@@ -405,6 +400,54 @@ instance DecodeAeson KupoScriptRef where
           flip note (convertNativeScript nativeScript) $
             TypeMismatch "decodeNativeScript: failed to convert native script"
 
+-------------------------------------------------------------------------------
+-- `isTxConfirmed` response parsing
+-------------------------------------------------------------------------------
+
+newtype KupoUtxoSlot = KupoUtxoSlot Slot
+
+derive instance Generic KupoUtxoSlot _
+derive instance Eq KupoUtxoSlot
+
+instance Show KupoUtxoSlot where
+  show = genericShow
+
+instance DecodeAeson KupoUtxoSlot where
+  decodeAeson = decodeAeson >>> map (slot >>> KupoUtxoSlot)
+    where
+    slot :: { created_at :: { slot_no :: Slot } } -> Slot
+    slot = _.created_at.slot_no
+
+unwrapKupoUtxoSlot :: KupoUtxoSlot -> Slot
+unwrapKupoUtxoSlot (KupoUtxoSlot slot) = slot
```

Comment:
Why not just add a `Newtype` instance?

Reply:
Wasn't really worth it, the type only exists to attach decoding instances to, if we use unwrap we have to give a type hint

Reply:
eg 

```
unwrapKupoUtxoSlot a
```
vs
```
unwrap (a :: KupoUtxoSlot)
```